src/CoreBundle/Service/MaintenanceModeService.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace ChameleonSystem\CoreBundle\Service;
+
+use ChameleonSystem\CoreBundle\Exception\MaintenanceModeErrorException;
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\DBALException;
+
+class MaintenanceModeService implements MaintenanceModeServiceInterface
+{
+    /**
+     * @var Connection
+     */
+    private $connection;
+
+    public function __construct(Connection $connection)
+    {
+        $this->connection = $connection;
+    }
+
+    public function isActivated(): bool
+    {
+        return file_exists(PATH_MAINTENANCE_MODE_MARKER);
+    }
+
+    public function isActivatedInDb(): bool
+    {
+        $col = $this->connection->fetchColumn("SELECT `shutdown_websites` FROM `cms_config`");
```

Comment:
Why not use `TdbCmsConfig::GetInstance()->fieldShutdownWebsites`?

Reply:
Because it references a Tdb

Reply:
Changed